static/style.css

```diff
@@ -23,76 +53,91 @@ a, a:link, a:visited {
 }
 
 a:hover {
- text-decoration: underline;
+	text-decoration: underline;
 }
 
 /* Header */
- header {
+
+header {
 	margin-top: 40px;
 	text-align: center;
 	color: #1182DB;
 }
- header > h1 {
+
+header > h1 {
 	font-size: 500%;
 	font-weight: bold;
 }
- header > h2 {
+
+header > h2 {
 	margin-left: 1em;
 	font-size: 200%;
 }
- header > section.warning {
+
+header > section.warning {
 	width: 40%;
 	background-color: #ffc600;
 	color: #5f5f5f;
 }
- header > section.critical-warning {
-	 width: 40%;
-	 background-color: #cf3e3e;
-	 font-weight: bold;
-	 color: white;
- }
 
-/* Input boxes */
- input[type="text"] {
+header > section.critical-warning {
+	width: 40%;
+	background-color: #cf3e3e;
+	font-weight: bold;
+	color: white;
+}
+
+select,
+input[type="text"],
+input[type="number"] {
 	background-color: white;
 	color: #404552;
 	border: 1px solid #dedede;
 	margin-left: 8px;
 	margin-bottom: 10px;
```

Comment:
The `margin-bottom` is not applied on `select` because on [line 26](https://github.com/RSS-Bridge/rss-bridge/pull/763/files#diff-dbe0876d6d2eb9f1d2b99bf53b34ece8R26) it was set to `0`.
`.parent element` is [more specific](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) than `element`.

Multiple property redefine is hard to maintain. Especially when we use shorthand property (like `margin`), it provides accidental CSS resets.

Reply:
Here all `margin` definition for `select` in a browser that support `grid`:

![image](https://user-images.githubusercontent.com/2306550/45459657-9bb14c80-b6f9-11e8-9446-e5e40987b5a0.png)

Reply:
You are right, I didn't even see that. Your solution above solves that quite nicely. :+1: